backend.rb

```diff
@@ -62,6 +62,7 @@ class Backend < Sinatra::Base
       article = Article.find(params[:article_id])
       return json data: article.paid_content
     else
+      status 500
```

Comment:
so the decoction is based on the status? if so you should update the integration Doc